src/components/Restaurant.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Restaurant = () => {
+  return (
+    <div>
+      <h3>Restaurant Orders</h3>
+      <ul>
+        <Order orderType={"Pizzas"} />
+        <Order orderType={"Salads"} />
+        <Order orderType={"Chocolate cake"} />
+      </ul>
+    </div>
+  );
+};
+
+const Order = ({ orderType }) => {
```

Comment:
The React convention is to split each component into it's own file. Could you move this to the appropriate place?